CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [8.2.31](https://github.com/BurdaMagazinOrg/thunder-distribution/tree/8.2.31) 2018-12-18
+[Full Changelog](https://github.com/BurdaMagazinOrg/thunder-distribution/compare/8.2.30...8.2.31)
+
+We fixed and improved several topics regarding the content moderation feature and started to reorganize the distribution 
+in preparation for the alpha release of Thunder 3.0. No new features have been added.
+
+- Fix [Cannot update to thunder 8.2.30 with composer](https://www.drupal.org/project/thunder/issues/3019353)
+- Fix [WSOD on scheduled nodes after saving](https://www.drupal.org/project/thunder/issues/3018530)
+- Fix [Wrong access check for revision reset](https://www.drupal.org/project/thunder/issues/3019596)
+- Fix [As a restricted editor I am not be able to edit scheduled nodes](https://www.drupal.org/project/thunder/issues/3020284)
```

Comment:
Wording issue: I am not be able to edit -> I am not able to
Although the ticket is named like this, it is confusing to understand